docs/Reference/Dropdown.md

```diff
@@ -306,7 +306,7 @@ var dropdown = new kintoneUIComponent.Dropdown({
 
 var body = document.getElementsByTagName("BODY")[0];
     body.appendChild(dropdown.render());
-
+var firstItem = dropdown.getItems()[0];
 dropdown.removeItem(0);
 console.log(firstItem);
```

Comment:
Need change:
console.log(firstItem); => console.log(dropdown.getItems());

Reply:
We no need to print all items. We just need to check the first item again when the old value is removed from list. 
This change is more suitable than all items printing.

Thank you

Reply:
You're right!